index.js

```diff
@@ -407,6 +407,72 @@ module.exports = {
         if (err) return cb(clarify(err, 'Error finding or creating additions feed when starting ssb-tribes2'))
         return cb()
       })
+
+      // look for new epochs that we're added to
+      ssb.metafeeds.findOrCreate((err, myRoot) => {
+        // prettier-ignore
+        if (err) return cb(clarify(err, 'Error getting own root in start()'))
+
+        pull(
+          ssb.db.query(
+            // TODO: does this output new stuff if we accept an invite to an old epoch and then find additions to newer epochs?
+            where(and(isDecrypted('box2'), type('group/add-member'))),
+            live({ old: true }),
+            toPullStream()
+          ),
+          // groups/epochs we're added to
+          pull.filter((msg) => {
+            return msg.value?.content?.recps?.includes(myRoot.id)
+          }),
+          // to find new epochs we only check groups we've accepted the invite to
+          paraMap((msg, cb) => {
+            pull(
+              ssb.box2.listGroupIds(),
+              pull.collect((err, groupIds) => {
+                // prettier-ignore
+                if (err) return cb(clarify(err, "Error getting groups we're already in when looking for new epochs"))
+
+                if (groupIds.includes(msg.value?.content?.recps?.[0])) {
+                  return cb(null, msg)
+                } else {
+                  return cb()
+                }
+              })
+            )
+          }),
```

Comment:
`paraMap(fn) === pull.asyncMap(fn)`
You need to add a "width" for it to be running in parallel e.g. run at most 4 at once:

```suggestion
          }, 4),
```

Reply:
done